### PR TITLE
feat(example):chat app example using boilerplate

### PR DESCRIPTION
**Pull request by RaghavaroraSF**
## Description
-  We are creating the chat-app using ARC/boilerplate.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yeshamavani** [comment]
rectify the commit message

File: projects/chat-app-example/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import {AuthGuard, LoggedInGuard} from '@project-lib/core/auth';
+
+const routes: Routes = [
+  {
+    path: 'auth',
+    loadChildren: () =>
+      import('projects/arc-lib/src/lib/components/auth/auth.module').then(
+        m => m.AuthModule,
+      ),
+   // canActivate: [LoggedInGuard],
```
Comment (yeshamavani) [inline projects/chat-app-example/src/app/app-routing.module.ts:13]: ??

**Comment by RaghavaroraSF** [comment]
> rectify the commit message

done

File: angular.json [hunk 2]
```diff
@@ -158,14 +298,17 @@
...
+              "zone.js",
+              "zone.js/testing"
+            ]
           }
         }
       }
     }
   },
-  "defaultProject": "arc",
+  "defaultProject": "chat-app-example",
```
[the remark this answers is not in the document]
Comment (RaghavaroraSF) [reply angular.json:310]: done

File: projects/chat-app-example/src/app/app-routing.module.ts (hunk 1 above)
Comment (RaghavaroraSF) [reply projects/chat-app-example/src/app/app-routing.module.ts:13]: resolved

**Pull request closed without merging**